### ISSUE TITLE
Blue bars on the sides on 4k display

### ISSUE THREAD

**Issue by Mangatt** [adobe/brackets#14464]
When I switch from brackets to another window and the switch back to brackets, blue bars on all sides appears.

This is how brackets looks when they starts:
![brackets-ok](https://user-images.githubusercontent.com/2706990/42266782-f1a31b64-7fa9-11e8-8be5-a322bac7eef5.png)

This is after window switch (notice thick bars at sides and bottom, thin at top, under menu):
![brackets-bars](https://user-images.githubusercontent.com/2706990/42266786-f557dede-7fa9-11e8-9c28-ee7148daf927.png)

I believe that might be caused somehow by 4k monitor, since on my other screen is everything just fine.

**Comment by navch** [comment]
I don't think this happens because of 4k. Even I noticed this on other machines. For me this usually happens when I connect multi-monitor.

**Comment by MerryPanda** [comment]
it happens always (win10, 1080p)...  probably brackets window drawing is a lil bit apart from win expectations
